docker/alazab-auth-manager/auth-manager.py

```
@@ -0,0 +1,290 @@
+            # Rollback
+            shutil.copy(backup_path, ENV_FILE)
+            self._apply_compose()
+            self._send_json({"error": "Failed to apply compose, rolled back"}, 500)
+            return
+
+        self._send_json({"status": "success", "revision": rev_id})
+
+    def _apply_compose(self) -> bool:
+        try:
+            # config check
+            subprocess.run(["docker", "compose", "--env-file", ".env", "--env-file", "auth.env", "config", "--quiet"], 
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)